examples/window-app/common/include/WindowApp.h

```diff
@@ -135,6 +135,20 @@ class WindowApp
         Timer * mTiltTimer            = nullptr;
         OperationalState mLiftOpState = OperationalState::Stall;
         OperationalState mTiltOpState = OperationalState::Stall;
+
+        struct CoverWorkData
+        {
+            chip::EndpointId mEndpointId;
+
+            union
+            {
+                chip::Percent100ths percent100ths;
+                OperationalStatus opStatus;
+            };
+        };
+
+        static void ScheduleTiltPositionSet(intptr_t arg);
```

Comment:
You did not use the same methodology for Lift and Tilt update
This is ok for me as long as it work, but why did you do 2 ways ? one with lock and the other with scheduling

The whole demo will be refactored along the way anyway  with other PRs for factorization and simplification

Reply:
Good catch, i updated the PR for lift and tilt to aligned.